### PR TITLE
Gconf removals

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

**Comment by q66** [comment]
why?

also, should turn into metapackage if it's really going to be removed

**Comment by Chocimier** [comment]
No, we do not want to force removing package from where it is installed by cheating that it will be updated.

**Comment by q66** [comment]
Considering it's a library unlikely to be directly used by anyone (added in 2015, never updated) and it's dead (and upstream removed it) we *do* want to remove this from users' systems or it'll cause problems at later point (when it goes incompatible with the environment).

File: srcpkgs/vscode/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'vscode'
 pkgname=vscode
 version=1.40.2
-revision=1
+revision=2
 hostmakedepends="pkg-config python nodejs-lts yarn"
 makedepends="libxkbfile-devel libsecret-devel"
-depends="GConf libXtst libxkbfile nss dejavu-fonts-ttf"
```
Comment (ericonr) [inline srcpkgs/vscode/template:7]: This has been applied already.

**Pull request closed without merging**